xchange-bitmex/src/main/java/org/knowm/xchange/bitmex/service/BitmexTradeService.java

```diff
@@ -187,7 +194,7 @@ public UserTrades getTradeHistory(TradeHistoryParams params) throws IOException
     }
 
     List<UserTrade> userTrades =
-        getTradeHistory(symbol, null, null, count, start, false, startTime, endTime).stream()
+        getTradeHistory(symbol, null, null, count, start, true, startTime, endTime).stream()
```

Comment:
this could potentially break downstream users expecting the oldest rows first, does it make more sense to add a param to `BitmexTradeHistoryParams` which uses could explicitly toggle and have the default set to something?

Reply:
maintains backward compat

    boolean reverse = (params instanceof TradeHistoryParamsSorted)
        && ((TradeHistoryParamsSorted) params).getOrder() == TradeHistoryParamsSorted.Order.desc;